specification/search/data-plane/Azure.Search/stable/2023-11-01/searchservice.json

```diff
@@ -9534,20 +9534,56 @@
     "SplitSkillLanguage": {
       "type": "string",
       "enum": [
+        "am",
```

Comment:
Why is there changes here when BYOE isn't the target of this release?

Reply:
This is because the increase in language support has existed on the SplitSkill for all API versions for over 2 years, and we just didn't remember to update the enum in the SDK until now. It's not new to BYOE.